### DISCUSSION TITLE
Rename dcline to simple_dc_link

### DISCUSSION THREAD

**Discussion by jkisse** [e2nIEE/pandapower#2462] [Ideas]
### Feature Checklist

- [X] Searched the [issues page](https://github.com/e2nIEE/pandapower/issues) for similar reports
- [X] Read the relevant sections of the [documentation](https://pandapower.readthedocs.io/en/latest/about.html)
- [X] Browse the [tutorials](https://github.com/e2nIEE/pandapower/tree/develop/tutorials) and [tests](https://github.com/e2nIEE/pandapower/tree/develop/pandapower/test) for usefull code snippets and examples of use

### Issue

The two components [`dcline`](https://pandapower.readthedocs.io/en/latest/elements/dcline.html) and [`line_dc`](https://pandapower.readthedocs.io/en/latest/elements/line_dc.html) can easily be confused and the documentation is not very precise in pointing out the difference between the two. I propose to rename "dcline" to "simple (HV)DC-link", as the electrical model apparently assumes converter stations as well.
The documentation and docstrings should be expanded and also mentioned, that dcline and line_dc are two completly different things and not intended to be used together.

### Label

- [ ] Relevant labels are selected

**Comment by vogt31337** [comment]
Really quite confusing, but I don't understand what a line_dc is? Where is it used and for which use case?
The dcline is modeled using generators but from the docs a line_dc seems to be a normal line? Is this for dc loadflows?!

**Reply by jkisse** [reply]
I think line_dc is intended to be used with [VSCs ](https://pandapower.readthedocs.io/en/latest/elements/vsc.html). It wouldn't make sense to use these converters with the dcline.
